src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -120,6 +120,22 @@ async def download_known_version(
         download_file = DownloadFile(url, FileDigest(known_version.sha256, known_version.filesize))
         return await Get(DownloadedExternalTool, ExternalToolRequest(download_file, exe))
 
+    package_managers = DictOption[str](
+        default={"npm": "8.5.5"},
```

Comment:
I would have preferred to set this to the empty dictionary, and let Corepacks own "knownGoodReleases.json" be in effect for npm by default as well.

Unfortunately that version (7.20.0) is not the same as the one that has been in use by pants so far.

Reply:
Maybe we can drop this default once corepack updates their default to 8.5.5 or later.